collect_app/src/main/java/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -107,12 +102,15 @@ public abstract class QuestionWidget extends FrameLayout implements Widget {
     @Inject
     ImageLoader imageLoader;
 
-    public QuestionWidget(Context context, QuestionDetails questionDetails) {
+    protected AudioPlayer audioPlayer;
+
+    public QuestionWidget(Context context, Dependencies dependencies, QuestionDetails questionDetails) {
```

Comment:
I think we'll hopefully move towards a world where changing this constructor isn't so awkward (#6534 is already pushing in the direction). For now, I felt like we should introduce an easier way to do it that doesn't rely on Dagger (which only works for application level dependencies) rather just adding `AudioPlayer` to every widget's contructor.